### PR TITLE
change to org.apache.pekko org id and update Github links to match apache/incubator-pekko-http

### PR DESCRIPTION
**Pull request by pjfanning**
Baby step toward switching from Akka to Pekko.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,23 +31,14 @@ The documentation is available at
...
-
-In addition to that, you may enjoy following:
-
-- The [news](https://akka.io/blog/news-archive.html) section of the page, which is updated whenever a new version is released
-- The [Akka Team Blog](https://akka.io/blog)
-- [@akkateam](https://twitter.com/akkateam) on Twitter
-- Projects built with Akka HTTP: [![Built with Akka HTTP][scaladex-badge]][scaladex-projects]
+- [GitHub discussions](https://github.com/apache/incubator-pekko-http/discussions): for questions and general discussion.
+- [Pekko dev mailing list](https://lists.apache.org/list.html?dev@pekko.apache.org): for Pekko development discussions.
+- [GitHub issues](https://github.com/apache/incubator-pekko-http/issues): for bug reports and feature requests. Please search the existing issues before creating new ones. If you are unsure whether you have found a bug, consider asking in GitHub discussions or the mailing list first.
```
Comment (mdedetrich) [inline README.md:39]: Shall we put a mention to the slack channel here as well?

Comment (mdedetrich) [inline README.md:39]: Shall we put a mention to the slack channel here as well?

**Review by mdedetrich — CHANGES_REQUESTED** [review]
I would add a reference to the pekko apache slack channel (you can also link to https://infra.apache.org/slack.html for instructions on how to signup to the slack channel)

Otherwise lgtm

File: README.md (hunk 1 above)
Comment (pjfanning) [reply README.md:39]: This is copied from main pekko repo.

Also, ASF Slack is for ASF committers - it isn't for anyone to just sign up too.

I's prefer to just to include the public means of communication.

Comment (mdedetrich) [reply README.md:39]: > I'd prefer to just to include the public means of communication.

That's odd, I am part of the ASF slack channel for Kafka and you don't need to be a committer to be in it (I'm not a Kafka committer).

Regardless I'll just approve the PR, we can figure this out later if possible.

**Review by mdedetrich — APPROVED** [review]
lgtm

**Pull request merged**